### DISCUSSION TITLE
How do I link Metaplex to my custom cloud server/hosting?

### DISCUSSION THREAD

**Discussion by TheHenn86** [metaplex-foundation/metaplex#1543] [Q&A]
Hi I need help linking my metaplex store to my cloud server/hosting since I don't want to use github hosting .
Any help would be greatly appreciated I've been trying to figure this out for days. My cloud hosting is on vultr.com.

**Comment by colinchen4** [comment]
Check the community docs https://artifact-staking.medium.com/setting-up-a-solana-metaplex-nft-storefront-in-the-cloud-a10ea2490ed9
